tests/SSHTunnelTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbExtractorSSHTunnel\Test;
+
+use Keboola\DbExtractor\Exception\UserException;
+use Keboola\DbExtractorLogger\Logger;
+use Keboola\DbExtractorSSHTunnel\SSHTunnel;
+use PHPUnit\Framework\TestCase;
+
+class SSHTunnelTest extends TestCase
+{
+    public function testConfig(): void
+    {
+        $dbConfig = [
+            'ssh' => [
+                'user' => 'root',
+                'sshHost' => 'sshproxy',
+                'sshPort' => '22',
+                'localPort' => '33306',
+                'keys' => ['private' => getenv('SSH_KEY_PRIVATE')],
+            ],
+            'host' => 'mysql',
+            'port' => '3306'
+        ];
+
+        $logger = new Logger('test');
+        $tunnel = new SSHTunnel($logger);
+        $newDbConfig = $tunnel->createSshTunnel($dbConfig);
+
+        $this->assertEquals(
```

Comment:
Tenhle test prosím přepiš tak aby assertoval prostě to co z toho má vypadnout, bez dbconfig. Navíc expected a actual jsou opačně.

Reply:
Nejdřív zkontroluj, že před tím, než ho upravíš, tak prochází! Abysme měli jistotu, že se úpravou nerozbije.

Reply:
> Tenhle test prosím přepiš tak aby assertoval prostě to co z toho má vypadnout, bez dbconfig. Navíc expected a actual jsou opačně.

bez dbConfig to nepůjde, protože to je require parametr - s tím configem to kontroluji abych ověřil že vylezou správný hodnoty (přepsaný na lokální ip a port)

Reply:
Jo, sorry,  já jsem se zmátnul tím, jak to bylo upačně expected a actual... sorry :)